src/main/java/uk/gov/hmcts/reform/roleassignment/domain/model/Assignment.java

```diff
@@ -45,6 +45,10 @@
     private JsonNode notes;
     private List<String> authorisations;
 
+    public void setStatus(Status status) {
```

Comment:
no need to create set method. As , we are using lombok annotation to handle these.

Reply:
It was introduced to help with debugging, thanks for spotting it.  Removed now